pipeline-steps/managed_identity.yaml

```diff
@@ -11,7 +11,7 @@ parameters:
 
 steps:
 
-  - template: keyvault-read.yaml
+  - template: steps/keyvault-read.yaml@cnp-azuredevops-libraries
```

Comment:
is this file managed_identity.yaml still needed?

Reply:
Not currently being used. We directly use `steps/terraform.yaml@cnp-azuredevops-libraries` in the Managed identity stage. Removed the file